### PR TITLE
Add pre-commit ci to config

### PR DESCRIPTION
**Pull request by Galdanwing**

### CODE REVIEW

File: .github/workflows/django.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [ '3.8', '3.9', '3.10' ]
+        python-version: ['3.10' ]
```
Comment (mehfluffy) [inline .github/workflows/django.yml:20]: can we have symmetric spacing please 🙃

**Review by mehfluffy — APPROVED** [review]

File: .github/workflows/django.yml (hunk 1 above)
Comment (Galdanwing) [reply .github/workflows/django.yml:20]: Will add pre-commit hooks that track yaml as well :+1:

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -27,23 +31,22 @@ repos:
       - id: isort
         name: isort-backend
         files: backend/
-        args: [ "--profile=black" ]
```
Comment (mehfluffy) [inline .pre-commit-config.yaml:30]: why is this removed?

File: backend/food/tests.py [hunk 3]
```diff
@@ -1,3 +1,40 @@
 from django.test import TestCase
 
+from food.models import Ingredient, Item, Recipe
+
 # Create your tests here.
+
+
+class RecipeTestCase(TestCase):
```
Comment (mehfluffy) [inline backend/food/tests.py:8]: I like pytest more than unittest :(
